visualization/app/codeCharta/util/scenarioHelper.spec.ts

```diff
@@ -17,7 +17,7 @@ describe("scenarioHelper", () => {
 
 	describe("importScenarios", () => {
 		it("should convert vectors when importing scenarios", () => {
-			scenarios[0].settings.appSettings.camera = { x: 0, y: 1, z: 2 } as any
+			scenarios[0].settings.appSettings.camera = { x: 0, y: 1, z: 2 } as Vector3
```

Comment:
```suggestion
			scenarios[0].settings.appSettings.camera = new Vector3(0,1,2)
```

Reply:
This test is intended to check that it works with regular objects as well.